### PR TITLE
add sync idle 7d+ users on discord button

### PR DESCRIPTION
**Pull request by anishpawaskar**
**Parent ticket:** https://github.com/Real-Dev-Squad/website-backend/issues/1522
**Closes:** https://github.com/Real-Dev-Squad/website-dashboard/issues/534

**What is change?**
Added sync idle 7d+ users on discord button. When super_user click on button it will add `group-idle-7d+` on users who are idle for more than 7 days. 

**Before:**
Dashboard _ Real Dev Squad - Google Chrome 30-09-2023 00_15_34
**After:**
Dashboard _ Real Dev Squad - Google Chrome 03-10-2023 08_43_43
**Test coverage:**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -75,6 +75,14 @@
         <div class="status" id="sync-idle-users-update"></div>
       </div>
 
+      <div class="button-container">
+        <button class="action-button" id="sync-idle-7d-Plus-users">
+          <span class="spinner"></span>
+          Sync Idle 7d+ Users On Discord
+        </button>
+        <div class="status" id="sync-idle-7d-Plus-users-update"></div>
```
Comment (Pratiyushkumar) [inline index.html:83]: why `div` tag is used instead of `p` tag?

File: script.js [hunk 2]
```diff
@@ -13,6 +13,10 @@ const syncNicknamesButton = document.getElementById(SYNC_NICKNAMES);
 const syncUsersStatusUpdate = document.getElementById(SYNC_USERS_STATUS_UPDATE);
 const syncIdleUsersButton = document.getElementById(SYNC_IDLE_USERS);
 const syncIdleUsersUpdate = document.getElementById(SYNC_IDLE_USERS_UPDATE);
+const syncIdle7dUsersButton = document.getElementById(SYNC_IDLE_7D_Plus_USERS);
```
Comment (Pratiyushkumar) [inline script.js:16]: why is this required if we are not using it in code

File: __tests__/home/home.test.js [hunk 3]
```diff
@@ -155,6 +187,45 @@ describe('Home Page', () => {
...
+      (element) => element.textContent,
+      latestSyncStatusElement,
+    );
+
+    expect(latestSyncStatusText).not.toBe(`Last Sync: Failed`);
+    expect(latestSyncStatusText).not.toBe(
+      `Last Sync: Synced Data Not Available`,
+    );
+    expect(latestSyncStatusText).not.toBe(`Last Sync: In progress`);
+  });
```
Comment (Pratiyushkumar) [inline __tests__/home/home.test.js:227]: NIT: is it possible to test the exact message that will be displayed on screen after idle user sync?

File: script.js (hunk 2 above)
Comment (anishpawaskar) [reply script.js:16]: im using it in a function `addClickEventListener` on line number 331 in same file and that SYNC_IDLE_7D_Plus_USERS is coming from `constant.js` file.

File: index.html (hunk 1 above)
Comment (anishpawaskar) [reply index.html:83]: if you look into code every buttons which are present in `index.html` is  following same format like they are using div for showing the status as you can see in below snippet this is another button
```suggestion
        <div class="button-container">
            <button class="action-button" id="sync-nicknames">
                <span class="spinner"></span>
                 Sync Users Nicknames
            </button>
        <div class="status" id="sync-nicknames-status-update"></div>
      </div>
```

**Review by MayurLund — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]
